DBusInterface.md

```diff
@@ -30,6 +32,9 @@ The **org.freedesktop.ratbag1.Device** interface provides:
 - Methods:
   - GetProfileByIndex(uint) -> returns the object path for the given index
   - Commit() -> commit the changes to the device
+  - GetSvg(string) -> returns the svg for the theme given, must be one of
```

Comment:
Maybe this should say that it returns an absolute path to the SVG, as string.

Also, since `ratbagd_device_get_theme_svg` can return an empty string (through [`goto out`](https://github.com/libratbag/libratbag/pull/209/files#diff-de4a098bea35a92dcc7b13868920af30R209)), maybe you should mention this as well.